### DISCUSSION TITLE
HX711  don't work properly  (Error found :))

### DISCUSSION THREAD

**Discussion by rapid1985** [arendst/Tasmota#15162] [Q&A] [answered]
Hello

I've already bought 2 pieces HX711 with 20kg but they don't work properly. I can calibrate it but the display isn't correct either 0kg or the weight of the calibration object isn't anymore. What am I doing wrong.
I've just bought a second piece of hardware and have the same problem, so I have no idea. I think it can only be a small thing.
the calibration object has 2610g on ESP32 Tasmota v11.0.0

thanks Alex 

![unknown](https://user-images.githubusercontent.com/50635785/158827024-51915202-92e5-4a72-852c-2c9c346d66fa.png)
![IMG_9474](https://user-images.githubusercontent.com/50635785/158827102-b656b700-48f3-4173-93be-4420f197ef8e.jpg)

**Comment by boozeman** [comment]
Let me quess! 

When the weight goes over 10kg, you get 0kg and WeightRaw value drops dramatically. If so, welcome on board :-) But if not, Check the pins.

Red E+
Black E-
White A-
Green A+

You can also use weighres command for showing g instead kg. (weightres 1, weightres 2, etc)

**Reply by rapid1985** [reply]
I checked the pins and they fit. I can't confirm that with the 10 kg

**Comment by arendst** [comment]
The cable connection as given above is OK. I use the same colors/connections.

I also suggest to mount the load cell differently.

The arrow pointing downwards is fine but it should be mounted to the plate NOT the base as you do.

The way you've mounted it results in negative values when a load is mounted. Tasmota will decline negative values and you won't see anything. Easy test: pull the plate up with some force; now you should see a value.

**Comment by rapid1985** [comment]
Hi,

I checked the pins and they fit. I can't confirm that with the 10 kg.

If I calibrate it with the reference weight (2610g), I can see if there is nothing on the scale
13:54:37.979 RSL: STATUS8 = {"StatusSNS":{"Time":"2022-03-18T13:54:37","HX711":{"Weight":0.0,"WeightRaw":566}," ESP32":{"Temperature":69.4},"TempUnit":"C"}}

if I put something on it no matter under the reference weight, it shows no change. If I put on more than the reference weight, it only gives me about 2610g.

very strange.
![wage2](https://user-images.githubusercontent.com/50635785/159007086-9dd435e6-c725-497a-a1fb-afc50b41ff6d.JPG)

Thanks

**Comment by arendst** [comment]
As said before, you've mounted the load cell upside down.

**Reply by rapid1985** [reply]
I also turned it around but there is no improvement? I can put the scales on both sides.
![waage3](https://user-images.githubusercontent.com/50635785/159007584-e90ab7bc-fe90-4f36-9f99-52c0bfdd46a8.jpg)

**Reply by rapid1985** [reply]
![waage gedreht](https://user-images.githubusercontent.com/50635785/159008630-430a8dfc-6a5e-4629-aecc-7cd42ad9009b.jpg)
Or have I thought incorrectly about that?

**Comment by arendst** [comment, marked as answer]
Yes you have.

In addition to have it up side down you also managed to mount the white plate directly to the loadcell. This makes it impossible for the loadcell to move horizontally.

**Reply by rapid1985** [reply]
now it fits i think.
now comes always calibrate error. I have to keep looking
![IMG_9488](https://user-images.githubusercontent.com/50635785/159016293-76d9ef5a-6615-4628-b3fd-776afd1eb8e8.jpg)

**Reply by rapid1985** [reply]
Error found, first possibly the wrong assembly ;) and second the 5V from the ESP after I resoldered the 3.3V it worked perfectly.
thank you for your help

**Comment by rapid1985** [comment]
Error found, first possibly the wrong assembly ;) and second the 5V from the ESP after I resoldered the 3.3V it worked perfectly.

thank you for your help
